core-bundle/src/EventListener/HttpCache/StripCookiesSubscriber.php

```diff
@@ -22,7 +22,7 @@
  */
 class StripCookiesSubscriber implements EventSubscriberInterface
 {
-    private const BLACKLIST = [
+    private const DISALLOW_LIST = [
```

Comment:
I don't like the naming. Haven't we agreed on using either allow/deny or include/excluded based on the context? Then this should be `DENY_LIST` (because we are using `$allowList` further down).